ArcGISRuntimeSDKQt_SampleViewers/ArcGISRuntimeSDKQt_Samples/qml/SampleListView.qml

```diff
@@ -101,6 +102,7 @@ Page {
                             drawer.close();
                             // launch sample...
                             SampleManager.currentSample = sample;
+                            GAnalytics.postEvent("sample_opened", {"sample name": sample.name, "referrer": "category list"});
```

Comment:
Should this also be behind an `if (GAnalytics.telemetryEnabled)` check? My previous recommendation would cover that (move state logic to the class itself).

Reply:
Yeah the state logic is built into the `postEvent` method